src/Sensors/WebChat/Events/FormEvent.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace actsmart\actsmart\Sensors\WebChat\Events;
+
+use actsmart\actsmart\Utils\Literals;
+use Ds\Map;
+
+class FormEvent extends WebChatEvent
+{
+    const EVENT_NAME = 'event.webchat.form';
+
+    private $userId = null;
```

Comment:
Can you please pull user id and timestamp into the parent WebChatEvent class (for this FormEvent and all webchat events)